README.md

```diff
@@ -9,10 +9,10 @@ Import into your dependenciesManagement sections like this (use the latest relea
 ```
    <dependencyManagement>
         <dependencies>
-            <!-- Secure Banking: Open Banking UK common dependencies -->
+            <!-- Secure API Gateway: Open Banking UK common dependencies -->
             <dependency>
-                <groupId>com.forgerock.securebanking.uk</groupId>
-                <artifactId>securebanking-openbanking-uk-bom</artifactId>
+                <groupId>com.forgerock.sapi.gateway.uk</groupId>
```

Comment:
groupId should be `com.forgerock.sapi.gateway` ?

Reply:
In that case is for artifacts regarding uk specs, the common artifacts have only `com.forgerock.sapi.gateway` but for the artifacts for uk spec it's nice to have a extra `uk` group

Reply:
Group id renamed to fix pattern for all artifacts `com.forgerock.sapi.gateway` with no extra groups